FordLabs/peoplemover
# Contact us page

## Issue
Resolves #803 

## What was done
- [x] Created a contact us page

## How to test
1. Go to any page in the app, scroll down, and click the contact us button in the footer. Ensure you're taken to the contact us page.
2. Try to submit the form without filling out all fields. Ensure you get shown a message saying to please fill out the empty fields.
3. Fill out the form fully and submit the form. Ensure that an api call to `/api/contact-us` gets made, the form clears, and the send button gets swapped out for a confirmation button and a link back to the dashboard.
